CONTRIBUTING.md

```diff
@@ -36,3 +36,22 @@ The main files are:
 9. `x64/` is the directory with all the machine code generation logic. It will be available in early July. Obviously this is the most complex part of the compiler. It defines a set of functions that translates assembly instructions to machine code, it builds complicated binaries from scratch byte by byte. It manually builds all headers, segments, sections, symtable, relocations, etc. Right now it only has basic support of the x64 platform/Mach-O format, and it can only generate `.o` files, which then have to be linked with `lld`.
 
 The rest of the directories are vlib modules: `builtin/` (strings, arrays, maps), `time/`, `os/`, etc. Their documentation is pretty clear.
+
+## Example Workflow for Contributing
+##### (provided by [@spytheman](https://github.com/spytheman))
+
+1. Clone https://github.com/vlang/v in a folder, say nv (`git clone https://github.com/vlang/v nv`)
+1. `cd nv`
+1. `git remote add pullrequest git@github.com:spytheman/v.git`  # (NOTE: this is my own forked repo of: https://github.com/vlang/v - After this, I just do normal git operations such as: `git pull` and so on.)
```

Comment:
`spytheman` in the urls may need to be changed to `YOUR_GITHUB_USERNAME` or something else more obviously intended to be customized by the user following the steps.